src/io/sb3/fromSb3.ts

```diff
@@ -423,6 +427,70 @@ export async function fromSb3JSON(json: sb3.ProjectJSON, options: { getAsset: Ge
     videoOn: stage.videoState === "on",
     videoAlpha: stage.videoTransparency
   });
+
+  const targetToBlocks: Map<Target, Array<Block>> = new Map();
+  for (const target of [project.stage, ...project.sprites]) {
+    targetToBlocks.set(target, flattenBlocks(target.scripts.flatMap(script => script.blocks)));
```

Comment:
What does `flatMap` do here? Does it concatenate every block in every script into one array?

Reply:
I'm wondering if this could be folded into `flattenBlocks`-- a `Script`'s `blocks` is a `Block[]`, and so is a C-block's substack(s).

Reply:
Oops, this is an issue I addressed in 560096ee42fec48fdea4f2ac1a2ecd9e0e55ba2b. I meant to include that on this pull request but forgot.